test/controllers/journeys/expenses/goodsToSellOrUse/DisallowableGoodsToSellOrUseAmountControllerSpec.scala

```diff
@@ -58,14 +54,7 @@ class DisallowableGoodsToSellOrUseAmountControllerSpec
       .success
       .value
 
-  private val mockSelfEmploymentService = mock[SelfEmploymentService]
-
-  override val bindings: List[Binding[_]] = List(
-    bind[ExpensesNavigator].toInstance(new FakeExpensesNavigator(onwardRoute)),
-    bind[SelfEmploymentService].toInstance(mockSelfEmploymentService)
-  )
-
-  when(mockSelfEmploymentService.persistAnswer(anyBusinessId, anyUserAnswers, any, any)(any)) thenReturn Future.successful(filledUserAnswers)
```

Comment:
Removing this works for now, as in the `ControllerSpec` we are mocking the session repository. I think ideally we would mock the service instead, so when the session repository mock gets removed in your TODO in `ControllerSpec`, we may need to replace it with a service mock.